test/e2e/nemo-dependencies/values.yaml

```diff
@@ -18,8 +18,8 @@ installation_namespace: nemo
 pvc:
   # Ignored when localPathProvisioner.enabled is true — in which case these will default to:
   # storage_class: "local-path" and volume_access_mode: "ReadWriteOnce"
-  storage_class: ""
```

Comment:
@mkhaas these will be ignored and set to those exact values when local-path-provisioner is enabled.